### ISSUE TITLE
ge, gt-type valdators do not work if declared in Config.fields in derived classes?

### ISSUE THREAD

**Issue by ydanilin** [samuelcolvin/pydantic#923]
* OS: **Ubuntu 18.04**
* Python version: **3.7.3 [GCC 7.3.0]**
* Pydantic version: **0.32.2**

Hello, I'm creating a data model with derived Pydantic classes to be used with FastApi framework and want to use `gt` validator for int.
1. In the first example, I declared it via "traditional" way, with `Schema` instance, everything works as expected:
```py
# Example 1
from pydantic import BaseModel, Schema

class ContractItem(BaseModel):
    sequence_no: int = Schema(..., ge=1)
    # more fields assumed, but omitted

class ContractItemCreate(ContractItem):
    subject: str

    class Config:
        fields = {
            'sequence_no': {'alias': 'SequenceNo'},  # aliases are needed by my project
            'subject': {'alias': 'Subject'},
        }

# let's try to break the ge validator:
print(ContractItemCreate(SequenceNo=0, Subject="Make a deposit"))
# Works as expected:  (type=value_error.number.not_ge; limit_value=1)
```
2. In the docs it is clearly mentioned: "Instead of using Schema, the `fields` property of the `Config` class can be used to set all the arguments above except default."  
If I try to declare `ge` in `Config.fields`:
```py
# Example 2
class ContractItemAnother(BaseModel):
    sequence_no: int

class ContractItemAnotherCreate(ContractItemAnother):
    subject: str

    class Config:
        fields = {
            'sequence_no': {'ge': 1, 'alias': 'SequenceNo'},
            'subject': {'alias': 'Subject'},
        }

# this time, 'ge' does not work:
print(ContractItemAnotherCreate(SequenceNo=0, Subject="Make a deposit"))
# Output: ContractItemAnotherCreate sequence_no=0 subject='Make a deposit'
```
**1. Interesting that without such inheritance (single class directly from BaseModel) `ge` in Config.fields works.**
**2. Aliases work fine in derived classes, but `ge`-type validators do not**
Basically I can survive with Example 1 but would rather stick to Example 2 due to more compact code.  
Are `ge` validators supposed to work if declared like in Example 2 or I'm doing something wrong?

**Comment by samuelcolvin** [comment]
This is fixed in v1 by #909.

If you can't upgrade to v1 immediately, best to use [`conint()`](https://5d584fcca7c9b70007d1c997--pydantic-docs.netlify.com/#exotic-types) for now.
